README.md

```diff
@@ -132,7 +132,7 @@ to the default security group, go to **EC2 Security Groups** in the AWS Manageme
 Console and search for the security group containing “ecs-cli”. Add a rule as
 described in the
 [Adding Rules to a Security Group](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html#adding-security-group-rule) topic.
-Alternatively, you may specify an existing security group ID with the
+Alternatively, you may specify the ID of one or more existing security groups with the
```

Comment:
Can you change this to "Alternatively, you may specify one or more existing security group ID ..."